tools/cachegrand-benchmark-generator/src/builder.c

```diff
@@ -24,12 +24,12 @@ section_t* builder_new_section_p() {
 bool builder_section_append_subsection(
         section_t *section,
         section_t *subsections) {
-    section->subsections = realloc(
-            section->subsections,
-            (section->n_subsections+1) * sizeof(section_t));
+    if (NULL != subsections) {
```

Comment:
In this way it's fine but in the future, you can try to improve it, returning straight away instead of having the main body of the function wrapped in the if so it will be more readable (the same applies to the other places that use a similar logic)

Reply:
Nono, u have absolutely right. I already fix that 😁

Reply:
:D